icu4c/source/test/intltest/units_data_test.cpp

```diff
@@ -60,21 +58,6 @@ void UnitsDataTest::testGetUnitCategory() {
     }
 }
 
-void UnitsDataTest::testGetAllConversionRates() {
```

Comment:
This looks like a data sanity test.  I don't know exactly why this test existed, but I think we should probably keep it.  However, we don't need to port it to Java.  It's okay to have tests like this one in C++ but not in Java.

Reply:
@sffc , `testGetUnitCategory` is added in this PR: https://github.com/unicode-org/icu/pull/1839

For `testGetAllConversionRates`, we already testing this in testing the `UnitsConverter`, and the test should be always for the public API, except we need to make more deep testing.

Reply:
I prefer that we keep the test only in C++ and add a comment explaining that.